app/pages/system/networking/IpPoolPage.tsx

```diff
@@ -73,7 +72,13 @@ export function IpPoolPage() {
   return (
     <>
       <PageHeader>
-        <PageTitle icon={<Networking24Icon />}>{pool.name}</PageTitle>
+        <PageTitle icon={<IpGlobal24Icon />}>{pool.name}</PageTitle>
+        <DocsPopover
+          heading="IP Pools"
+          icon={<IpGlobal16Icon />}
+          summary="IP pools are collections of external IP addresses that let you control which silos can use which IPs."
+          links={[docLinks.systemIpPools]}
+        />
```

Comment:
I think this is a little wordy, but works …
<img width="1221" alt="Screenshot 2024-04-30 at 10 57 22 AM" src="https://github.com/oxidecomputer/console/assets/22547/1b060d0e-4631-483a-a8db-d3e6c01e996d">

Reply:
Yeah, I think it's better to have a bit more info despite the length, and we can make it a bit shorter. I went back and forth thinking about saying different things on IP pools list and IP pool detail, but I think this would be fine and works for both.

> IP pools are collections of external IPs you can assign to silos. When a pool is linked to a silo, users in that silo can allocate IPs from the pool for their instances.

![Screenshot 2024-04-30 at 2 23 43 PM](https://github.com/oxidecomputer/console/assets/3612203/c74937d8-98a5-4517-b4d8-2ca935b42cfa)

I think the default line is a bit much for the popover, but what if we stuck it in a tooltip on the default column? That column header is ridiculous anyway, it demands explanation.

Reply:
Added that tooltip to the table.